index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>John Thiong'o Murianki</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <section class ="content">
+    <header>
+        <a id="logo">
+            <img src="/media/My Logo.png">
```

Comment:
- **[OPTIONAL]** It looks like the image URL is broken as it is not being displayed. I highly recommend fixing this issue will improve your UI. 💯 
- **[OPTIONAL]** The hamburger menu is not aligned properly and does not match the Figma template. I would also recommend fixing this issue. 😄 

YOUR DESIGN
<img width="374" alt="Screenshot 2023-01-19 at 4 34 43 PM" src="https://user-images.githubusercontent.com/24832846/213426361-1bb0fc04-33c9-4fb8-b238-c4d97f6c262e.png">

FIGMA DESIGN
<img width="431" alt="Screenshot 2023-01-19 at 4 36 17 PM" src="https://user-images.githubusercontent.com/24832846/213426669-1e64d70d-8b71-4263-846b-0b5f6afc8607.png">